src/main/java/it/gov/pagopa/rtd/ms/rtdmsfilereporter/telemetry/AppInsightConfig.java

```diff
@@ -46,4 +48,10 @@ public MongoClientSettingsBuilderCustomizer mongoOpenTelemetryBridge(
     return clientSettingsBuilder -> clientSettingsBuilder
         .addCommandListener(MongoTelemetry.builder(openTelemetry).build().newCommandListener());
   }
+
+  @Bean
+  public HttpClientBuilder createBuilder(OpenTelemetry openTelemetry) {
```

Comment:
i think this is redundant with the @Bean injected into `ApacheHttpClient5Configuration.java`